nixos/modules/services/security/kloak.nix

```
@@ -0,0 +1,79 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.kloak;
+in
+{
+  options.services.kloak = {
+    enable = lib.mkEnableOption "kloak, a keystroke-level online anonymization kernel";
+
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = pkgs.kloak;
+      defaultText = lib.literalExpression "pkgs.kloak";
+      description = "The kloak package to use.";
+    };
```

There's an abstraction that does exactly that:
```suggestion
    package = lib.mkPackageOption pkgs "kloak" { };
```